examples/pytorch/02-start-with-dask.ipynb

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
<p>the cluster part should be <code>SaturnCluster(n_workers=n_workers)</code>  just in case the user didnt start the cluster from the UI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>